citar-filenotify.el

```diff
@@ -79,8 +79,8 @@ If it is other than 'global or 'local invalidate both"
 
 CHANGE refers to the notify argument."
   (pcase (cadr change)
-    ('(nil changed) (funcall func scope))
-    ('(created deleted renamed)
+    ((or (pred null) 'changed) (funcall func scope))
```

Comment:
(or 'nil 'change) would work too.

Reply:
I don't understand pcase very well either @andersjohansson. 

Can you take a look at @minad's suggestion see WHYT?

Reply:
That looks better, pushed that!